modules/ROOT/pages/7.0.1-release-notes.adoc

```diff
@@ -60,6 +60,21 @@ For information on the **<Open source plugin name>** plugin, see xref:<plugincod
 
 The following premium plugin updates were released alongside {productname} {release-version}.
 
+=== Markdown
+
+The {productname} {release-version} release includes an accompanying release of the **Markdown** premium plugin.
+
+This **Markdown** release includes the following fixes and improvements.
+
+==== Pasting markdown into some situations would cause the pasted data to be placed in unexpected places.
+// #TINY-10766
+
+Previously in the **Markdown** plugin, the "markdown-on-paste" feature used the `editor.selection.setContent` function to insert content into the {productname} editor. However, this approach resulted in unexpected cursor behavior, undo levels, and formatting issues.
```

Comment:
Do we need to show these particular details? While that is what the actual change is, it might be easier to understand if we put it as some sort of "Previous content insertion was not given the expected formatting treatment" or something along those lines?
If we're fine with those details being here though, seems fine.

Reply:
I feel like these details are suitable, and would suggest merging this as they are personally.